utils/utils.go

```diff
@@ -122,6 +125,54 @@ func FilePath(name, ext string, escape bool) (string, error) {
 	return outputPath, nil
 }
 
+// FileLineCounter Counts line in file
+func FileLineCounter(r io.Reader) (int, error) {
```

Comment:
this function seems useless for now

Reply:
It very usefull for testing! Should I delete it anyways?

Reply:
oh you can keep it